src/pages/Main/Components/MainSection/MainSection.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import MainSectionMovieList from '../MainSectionMovieList/MainSectionMovieList';
 import './MainSection.scss';
-
 export default class MainSection extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      movieInfomationList: [],
+    };
+  }
+  componentDidMount() {
+    fetch('http://localhost:3000/data/MainMockData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(movieInfomation => {
+        this.setState({
+          movieInfomationList: movieInfomation,
+        });
+      });
```

Comment:
인자명을 수정하고 객체 단축 속성명을 활용하시면 좀 더 깔끔하게 보여질 수도 있습니다.
```suggestion
      .then(movieInfomationList => {
        this.setState({
          movieInfomationList
        });
      });
```

Reply:
오... 수정했습니다 ! 좋은 정보 감사합니다